Resources/Prototypes/_Goobstation/Loadouts/loadout_groups.yml

```
@@ -1044,6 +1044,7 @@
   - JackBoots
   - BlackCowboyBoots
   - SnowjacksBso
+  - BSOAdvancedJackBoots
```

```suggestion
  - BSOAdvancedJackBoots #Omu
```
Comment any changes outside of _Omu